packages/amd-loader/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@polymer/amd-loader",
```

Comment:
We might want a name that indicates that this isn't full AMD: `amd-path-loader`?